WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2322,3 +2322,19 @@ PrefOOOExternalMsg          = margin-top:@PrefViewGutterSpace@;
 PrefOOOTimePeriodCell       = padding:@PrefSectionInternalSpacing@ 5px;
 PrefOOOMiniDropDown         = @PrefRowMiniSelectWidget@;
 PrefOOODurationLabel        = text-transform:lowercase;
+
+######################################
+#   PREFERENCES -Mail Page
+######################################
+
+PrefMailComposeRowValueWrapperWidth            = 22%
+PrefMailBlockMessageCellWidth                  = 55%
+PrefMailComposeRowSelectWidgetWidth            = 130px
+PrefMailComposeRowColorPickerWidgetWidth       = 1%
+PrefMailComposeLabel                           = padding-top:@SkinWrapperPadding@
+PrefMailComposeRowLabel                        = padding:11px @SkinViewInnerSpacing@; vertical-align:top
```

Comment:
As discussed having 12px moves the label bit down, for this I have used 11px.
using PrefViewInnerSpacing now.
https://github.com/Zimbra/zm-web-client/pull/102/commits/58bc858a52d6225549351a9fb2dbb96ad0f1ac08